FEM/rf_mmp_new.h

```diff
@@ -311,6 +317,60 @@ class CMediumProperties
 	double graindiameter;
 	double hydraulicrad;
 	double betaexpo;
+    // HS: borehole heat exchanger related parameters
```

Comment:
Can you put all these data to a stuct data, and define the struct in a single file? If possible to have a single class to handle these data the their associated methods.